e2e-polybft/e2e/jsonrpc_test.go

```diff
@@ -0,0 +1,69 @@
+package e2e
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+/*
+contract Sample {
+	uint256 val;
+
+	function getValue() public returns (uint256) {
+		return val;
+	}
+}
+*/
+
+var simpleCallContract = "608060405234801561001057600080fd5b5060c38061001f6000396000f3fe6080604052348015600f57600080fd5b506004361060285760003560e01c80632096525514602d575b600080fd5b60336047565b604051603e9190605d565b60405180910390f35b60008054905090565b6057816076565b82525050565b6000602082019050607060008301846050565b92915050565b600081905091905056fea365627a7a7231582071227b651950047ffed0fb11e7717a1fab7eb241a8944de583d00722050112596c6578706572696d656e74616cf564736f6c63430005110040"
```

Comment:
To me, it would be better to put this test contract to the `consensus/polybft/contractsapi/test-contracts`, the same way we did for `TestL1StateReceiver`.

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1337/commits/788a5857600d952439f5d4b0b0f2fa16b13c5de7